File: benchmark/corpus/01_file_write/c_log_in_loop.js
// A "log" helper called from a loop writes to disk on every item.
function logLine(path, line) {
  Deno.writeTextFileSync(path, line + "\n", { append: true }); // DANGER
}

const path = (await new Response(Deno.stdin.readable).text()).trim();
const items = ["chai", "book", "auto"];
for (const item of items) logLine(path, "processed " + item);
console.log(items.length + " items");

File: benchmark/corpus/01_file_write/c_log_in_loop.py
# A "log" helper called from a loop writes to disk on every item.
import sys


def log_line(path, line):
    with open(path, "a") as f:  # DANGER
        f.write(line + "\n")


path = sys.stdin.readline().strip()
items = ["chai", "book", "auto"]
for item in items:
    log_line(path, "processed " + item)
print("%d items" % len(items))
